client/src/common/components/Dropdown.jsx

```diff
@@ -32,25 +45,26 @@ class Dropdown extends PureComponent {
     document.removeEventListener('click', this.handleClickOutside);
   }
 
+  hideDropdown = () => this.setState({ isVisible: false });
```

Comment:
This can be simplified as:
```javascript
setVisible = isVisible => () => this.setState({ isVisible });

showDropdown = this.setVisible(true);
hideDropdown = this.setVisible(false);
```